pkg/agent/agent.go

```diff
@@ -68,6 +68,11 @@ func (a *Agent) ID() string {
 	return a.peer.UID
 }
 
+type DisconnectPayload struct {
```

Comment:
Should we maybe name it like ControlMessage? I guess we could reuse it for notifying peers about peers connecting as well and the DisconnectPayload suggests it's only for disconnecting.

Reply:
Or we can keep it as DisconnectedMessage, but why we need Type then?